docker-compose.yml

```
@@ -87,6 +87,8 @@ services:
       - ./htmlcov:/opt/warehouse/src/htmlcov:z
       - ./setup.cfg:/opt/warehouse/src/setup.cfg:z
       - ./pyproject.toml:/opt/warehouse/src/pyproject.toml:z
+      - ./.coveragerc:/opt/warehouse/src/.coveragerc:z
+      - ./sitecustomize.py:/opt/warehouse/src/sitecustomize.py:z
```

Could benefit from a comment, since site customization is usually a bit of a hack:

```suggestion
      # Site customization shim to enable multi-process coverage collection in tests.
      # See: https://coverage.readthedocs.io/en/latest/subprocess.html
      - ./sitecustomize.py:/opt/warehouse/src/sitecustomize.py:z
```